app/api/menu_routes.py

```diff
@@ -0,0 +1,61 @@
+from flask import Blueprint, request, jsonify
+from app.models import db, Menu, Business
+from ..forms.menu_form import MenuForm
+from flask_login import current_user, login_required
+from .auth_routes import validation_errors_to_error_messages
+
+menu_routes = Blueprint("menus", __name__, url_prefix="/menus")
+
+#Get All Menus
```

Comment:
Get menus of a business instead of all menus. 

Should be moved to business routes.